classes/plugins/ImportExportPlugin.php

```diff
@@ -412,26 +425,35 @@ public function getBounceTab($request, $title, $bounceUrl, $bounceParameterArray
      *
      * @param string $exportFileName
      */
-    public function downloadExportedFile($exportFileName)
+    public function downloadExportedFile(string $exportFileName, string $exportFilePath, PKPImportExportDeployment $deployment)
     {
+        $date = DateTime::createFromFormat(self::EXPORT_FILE_DATE_PART_FORMAT,  $exportFilePath);
+        if (!$date) {
+            return false;
+        } 
+
+        $exportFileName = $this->getExportFileName($this->getExportPath(), $exportFileName, $deployment->getContext(), '.xml', $date);
         $fileManager = new FileManager();
         $fileManager->downloadByPath($exportFileName);
         $fileManager->deleteByPath($exportFileName);
+        
+        return true;
     }
 
     /**
      * Create file given it's name and content
      *
      * @param string $filename
      * @param string $fileContent
-     * @param Context $context
+     * @param \Context $context
```

Comment:
Best not to refer to classes in the root namespace, since we'll be removing the aliases sometime in the future. Here, for example, it should be `\PKP\context\Context`. (Also below.)

Reply:
Fixed